src/front/app/components/globals/Button.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+const classNames = require('classnames');
+
+const buttonTypes = [
+  'primary',
+  'secondary',
+  'success',
+  'danger',
+  'warning',
+  'info',
+  'light',
+  'dark',
+  'link',
+];
+
+class Button extends Component {
+  static propTypes = {
+    buttonType: PropTypes.oneOf(buttonTypes).isRequired,
+    className: PropTypes.string,
+    children: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.node,
+      PropTypes.instanceOf(Component),
+    ]),
+    href: PropTypes.string,
+    to: PropTypes.string,
+  };
+
+  static defaultProps = {
+    buttonType: 'primary',
+    children: 'Click Me!',
```

Comment:
No, children is hard required, we don't want default values.

Reply:
But you said specifically to add default values..?